src/Atomic/pages/ApiLoadingPage.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useContext} from 'react';
+import {
+  ActivityIndicator,
+  View,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+import {Icon, Layout, Text} from '@ui-kitten/components';
+import globalStyles, {monofontFamily, colorGreen} from 'src/styles';
+import ScreenNavigation from 'layout/ScreenNavigation';
+import NetworkItem from 'presentational/NetworkItem';
+import {NetworkContext} from 'context/NetworkContext';
+import {NavigationProp} from '@react-navigation/native';
+import {ChainApiContext} from 'context/ChainApiContext';
+
+function LoadingView({navigation}: {navigation: NavigationProp<any>}) {
+  const {currentNetwork} = useContext(NetworkContext);
+  const {api} = useContext(ChainApiContext);
+
+  /// navigate to app in case of successfully connecting
+  const prevApi = React.useRef(!!api);
+  React.useEffect(() => {
+    if (api && !prevApi.current) {
+      navigation.navigate('App');
+    }
+  }, [navigation, api]);
+
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
what is this?

Reply:
I am used to separating stuff with a line of comment like this. I will remove them ✅